linode/resource_linode_object_storage_bucket.go

```diff
@@ -129,6 +162,28 @@ func resourceLinodeObjectStorageBucketDelete(d *schema.ResourceData, meta interf
 	return nil
 }
 
+func updateLinodeObjectStorageBucketAccess(d *schema.ResourceData, client linodego.Client) error {
+	cluster := d.Get("cluster").(string)
+	label := d.Get("label").(string)
+
+	updateOpts := linodego.ObjectStorageBucketUpdateAccessOptions{}
+	if d.HasChange("acl") {
+		updateOpts.ACL = linodego.ObjectStorageACL(d.Get("acl").(string))
+	}
+
+	if d.HasChange("cors_enabled") {
+		newCorsBool := d.Get("cors_enabled").(bool)
+		updateOpts.CorsEnabled = &newCorsBool
+	}
+
+	err := client.UpdateObjectStorageBucketAccess(context.Background(), cluster, label, updateOpts)
```

Comment:
since `err` isn't used outside of the `if` block, I'd do `if err := Func(); err != nil { ... }`.